### DISCUSSION TITLE
How to add the official example cloud of three.js to itwons

### DISCUSSION THREAD

**Discussion by GGBond-GIS** [iTowns/itowns#1839] [Q&A]
![image](https://user-images.githubusercontent.com/92559206/162730507-20d6876f-15f9-450e-b3e0-6f3d59b902d5.png)
![mbd_file_OTQ2MzEzXzIzNjkwOF8xNjQ3NTk5MTgyMzY5XzE2NDc1OTkxODIzNzE](https://user-images.githubusercontent.com/92559206/162730806-176933f7-efe6-4dd9-bb0c-3a521dae6b45.gif)
like this

**Comment by gchoqueux** [comment]
add the cloud to `view.scene` and initialize the `cloud.position` with a `Coordinates` object
